cmd/tk/export.go

```diff
@@ -29,13 +29,17 @@ func exportCmd() *cli.Command {
 
 	vars := workflowFlags(cmd.Flags())
 	getExtCode := extCodeParser(cmd.Flags())
+	allowSubFolderCreation := cmd.Flags().Bool("allow-subfolder-creation", false, "Allow export to create subfolders in the out directory if the format contains '/'")
```

Comment:
This flag is fairly long, I suggest instead:

```suggestion
	createDirs := cmd.Flags().Bool("dirs", false, "Treat '/' in filenames as directories")
```